### PR TITLE
Chore/poetry 3.0.0

### PR DESCRIPTION
**Pull request by Vixtir**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,18 +1,34 @@
...
+RUN poetry config virtualenvs.create false
+RUN poetry config experimental.new-installer false
 
-COPY . ./
+COPY poetry.lock pyproject.toml ./
+RUN poetry install --no-interaction --no-ansi -vvv
+
+
+COPY version version
+COPY .git .git
```
Comment (akastav) [inline Dockerfile:25]: COPY .git/ ./.git/

File: hydro_stat/app.py [hunk 2]
```diff
@@ -14,14 +14,15 @@
...
+fileConfig("hydro_stat/resources/logging_config.conf")
+
 hs_cluster = Cluster(HTTP_UI_ADDRESS)
 flask_app = Flask(__name__)
 CORS(flask_app)
 
 if DEBUG_ENV:
     log = logging.getLogger('werkzeug')
-    log.setLevel(logging.DEBUG)
+    logging.setLevel(logging.DEBUG)
```
Comment (KineticCookie) [inline hydro_stat/app.py:25]: `log` or `logging`?

File: tests/test_heatmap.py [hunk 3]
```diff
@@ -7,7 +7,7 @@
 import pytest
 from sklearn.preprocessing import LabelEncoder
 
-from app.statistical_report.statistical_feature_report import HeatMapData
+from statistical_report.statistical_feature_report import HeatMapData
```
Comment (KineticCookie) [inline tests/test_heatmap.py:10]: Import without correct root package

File: tests/test_report.py [hunk 4]
```diff
@@ -5,8 +5,8 @@
 import pytest
 from hydrosdk import Cluster, ModelVersion
 
-from statistical_report.statistical_feature_report import HeatMapData
-from statistical_report.statistical_report import StatisticalReport
+from app.statistical_report.statistical_feature_report import HeatMapData
+from app.statistical_report.statistical_report import StatisticalReport
```
Comment (KineticCookie) [inline tests/test_report.py:9]: IIRC, `app` is not the correct package name but `hydro_stat`

**Review by KineticCookie — CHANGES_REQUESTED** [review]

**Pull request merged**